File: src/main/kotlin/com/xs0/gqlktx/schema/intro/GqlIntroDirectiveLocation.java
package com.xs0.gqlktx.schema.intro;

import com.xs0.gqlktx.ann.GraphQLEnum;

@GraphQLEnum("__DirectiveLocation")
public enum GqlIntroDirectiveLocation {
    QUERY,
    MUTATION,
    SUBSCRIPTION,
    FIELD,
    FRAGMENT_DEFINITION,
    FRAGMENT_SPREAD,
    INLINE_FRAGMENT
}

File: src/main/kotlin/com/xs0/gqlktx/schema/intro/GqlIntroDirectiveLocation.kt
package com.xs0.gqlktx.schema.intro

import com.xs0.gqlktx.GqlEnum

@GqlEnum("__DirectiveLocation")
enum class GqlIntroDirectiveLocation {
    QUERY,
    MUTATION,
    SUBSCRIPTION,
    FIELD,
    FRAGMENT_DEFINITION,
    FRAGMENT_SPREAD,
    INLINE_FRAGMENT
}
